site/README.md

```diff
@@ -0,0 +1,137 @@
+# NGINX Gateway Fabric Documentation
+
+This directory contains all of the user documentation for NGINX Gateway Fabric, as well as the requirements for building and publishing the documentation.
+
+Documentation is written in Markdown, built using [Hugo](https://gohugo.io) and deployed with [Netlify](https://www.netlify.com/).
+
+## Setup
+
+Hugo is the only requirement for building documentation.
+
+To install Hugo locally, follow the [official Hugo instructions](https://gohugo.io/getting-started/installing/).
+
+> **Note**: We are currently running [Hugo v0.115.3](https://github.com/gohugoio/hugo/releases/tag/v0.115.3) in production.
+
+If you have [Docker](https://www.docker.com/get-started/) installed, there is a fallback in the [Makefile](Makefile) which means you do need to install Hugo locally.
+
+## Developing documentation locally
+
+To build the docs locally, run the `make` command inside this `/docs` directory:
+
+```text
+make clean          -   Removes the local `public` directory, which is the default output path used by Hugo
```

Comment:
Do we need the backticks within a code block? They don't render so, it just looks like this:
```
`public`
```